darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1487,14 +1484,17 @@ def predict_from_dataset(
             mc_dropout=mc_dropout,
         )
 
+        if data_loader_kwargs is None:
```

Comment:
Shouldn't we allow then here as well the liberty to overwrite these defaults (expect shuffle)?